views/admin/network-updates.php

```diff
@@ -0,0 +1,59 @@
+<?php extract( $data ); ?>
+<div class="wrap">
+	<h2><?php _e( 'Jetpack Network Update', 'jetpack' ); ?></h2>
+</div>
+
+<?php
+
+$new_nonce = wp_create_nonce( 'jetpack-network-update' );
+
+if ( $action = 'update' && wp_verify_nonce( $nonce, 'jetpack-network-update' ) ) {
+	$next_page = $num + 1;
+	$size = 2;
+	$sites = get_sites( array( 'offset' => $num * $size, 'number' => $size ) );
+	$done = sprintf( __( 'We sucessfully updated Jetpack to %s across your network.','jetpack' ), JETPACK__VERSION );
```

Comment:
I think a translator comment would be useful here, so translators know that we expect a number here.